libr/debug/p/debug_gdb.c

```diff
@@ -12,6 +12,13 @@ typedef struct {
 #define UNSUPPORTED 0
 #define SUPPORTED 1
 
+#define PROC_NAME_SZ   1024
```

Comment:
Are those constants actually coming from the kernel? If not i would prefer to have no limits on any of this